File: Operacoes/sobrou.cpp
/*@sobrou: Calculando quanto sobrou*/

#include <iostream>
#include <iomanip>
#include <locale>
using namespace std;

int main(){
    setlocale(LC_ALL, "Portuguese");
    int qtd1, qtd2, qtd3;
    double p1, p2, p3;
    double qtdDinheiro;

    cin >> qtd1;
    cin >> qtd2;
    cin >> qtd3;
    cin >> p1;
    cin >> p2;
    cin >> p3;
    cin >> qtdDinheiro;

    double total = (qtd1 * p1) + (qtd2 * p2) + (qtd3 * p3);
    double troco = qtdDinheiro - total;
    cout << fixed << setprecision(2);
    cout << troco << "\n";
    return 0;
}

File: src/Operacoes/Sobrou.java
/**
 * Questão: @sobrou: Calculando quanto sobrou
 *
 * @author UxieGu1
 */
package Operacoes;
import java.util.Scanner;

public class Sobrou {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int qtd1 = scanner.nextInt();
        int qtd2 = scanner.nextInt();
        int qtd3 = scanner.nextInt();
        double p1 = scanner.nextDouble();
        double p2 = scanner.nextDouble();
        double p3 = scanner.nextDouble();
        double qtdDinheiro = scanner.nextDouble();

        double total = (qtd1 * p1) + (qtd2 * p2) + (qtd3 * p3);
        double troco = qtdDinheiro - total;

        System.out.printf("%.2f%n", troco);
        scanner.close();
    }
}
